keel-sql/src/main/resources/db/changelog/20200122-service-account-on-delivery-config.yml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+- changeSet:
+    id: service-account-on-delivery-config
+    author: fletch
+    changes:
+    - addColumn:
+        tableName: delivery_config
+        columns:
+        - column:
+            name: service_account
+            type: varchar(255)
```

Comment:
are you going to have a default value here, or do something that will insert the correct  values?

Reply:
Working on it right now